### ISSUE TITLE
Pin Issues with 10.3.3

### ISSUE THREAD

**Issue by Ky-0** [jellyfin/jellyfin#1386]
Just updated to JF 10.3.3 and I cannot set a PIN, when I try to and hit save I just get the loading wheel that does nothing (stays there loading). However the easy sign in seems to work now as was broken prior when the pin was left blank and easy sign in enabled wouldnt work previously which it does now - but you cannot set a pin so cant verify if the pin login works once one has been set.

Config:
Debian 9 - REPO Install for JF
Chrome Browser [Version 74.0.3729.157]

**Comment by anthonylavado** [comment]
Merging from other issue [(original)](https://github.com/jellyfin/jellyfin/issues/1392#issue-446074891):
**Describe the bug**
After update to 10.3.3 i wanted to try fixed pin login. But after setting a pin no page can be fully loaded.

**To Reproduce**
Set a pin

**Expected behavior**


**Logs**
```
[11:44:31] [ERR] Error processing request
jellyfin_1           | System.IndexOutOfRangeException: Index was outside the bounds of the array.
jellyfin_1           |    at MediaBrowser.Model.Cryptography.PasswordHash..ctor(String storageString)
jellyfin_1           |    at Emby.Server.Implementations.Library.UserManager.GetLocalPasswordHash(User user)
jellyfin_1           |    at Emby.Server.Implementations.Library.UserManager.GetUserDto(User user, String remoteEndPoint)
jellyfin_1           |    at System.Linq.Enumerable.SelectIPartitionIterator`2.LazyToArray()
jellyfin_1           |    at System.Linq.Enumerable.SelectIPartitionIterator`2.ToArray()
jellyfin_1           |    at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
jellyfin_1           |    at MediaBrowser.Api.UserService.Get(GetUsers request, Boolean filterByDevice, Boolean filterByNetwork)
jellyfin_1           |    at Emby.Server.Implementations.Services.ServiceExecGeneral.Execute(Type serviceType, IRequest request, Object instance, Object requestDto, String requestName)
jellyfin_1           |    at Emby.Server.Implementations.Services.ServiceController.Execute(HttpListenerHost appHost, Object requestDto, IRequest req)
jellyfin_1           |    at Emby.Server.Implementations.Services.ServiceHandler.ProcessRequestAsync(HttpListenerHost appHost, IRequest httpReq, IResponse httpRes, ILogger logger, String operationName, CancellationToken cancellationToken)
jellyfin_1           |    at Emby.Server.Implementations.HttpServer.HttpListenerHost.RequestHandler(IHttpRequest httpReq, String urlString, String host, String localPath, CancellationToken cancellationToken)
```


**System (please complete the following information):**
 - OS: Docker under Linux
 - Browser: Chrome
 - Jellyfin Version: 10.3.3
 - Reverse proxy: traefik

**Comment by mattydebie** [comment]
I'm having the same error as @anthonylavado followed by a 
```
[ERR] Could not find handler for /embywebsocket
```

**Comment by anthonylavado** [comment]
Okay, there's a suggested fix here: https://github.com/jellyfin/jellyfin/pull/1372#issuecomment-494272833

Just need to implement it...

**Comment by DavHau** [comment]
Thanks for fixing it. But is there any way to disable the pin somehow without accessing the web interface? Many users (including me) won't be able to access their jellyfin until the new version is released. Is it safe for me to compile the 10.3.z branch and replace my current instance with it or will i risk corrupting my data?

**Comment by anthonylavado** [comment]
@DavHau If you reset a user's password, that will clear it.

Go to http://MyJellyfinServer:8096web/index.html#!/forgotpassword.html

**Comment by DrPandemic** [comment]
For some reason this issue didn't get auto closed by my PR... Let's close it manually. The fix will be present in the next release 10.3.4
